components/buttons/buttons.css

```diff
@@ -0,0 +1,130 @@
+@import url(/stylesheets/default.css);
+
+main header {
+  height: 100%;
+}
+.buttons-link {
+  background-color: var(--extra-color);
+}
+
+/* buttons css */
+
+/* primary */
+.primary-buttons button {
+  font-size: 1.3rem;
+  color: black;
+  padding: var(--gap-small);
+  margin: var(--gap-medium);
+}
```

Comment:
very nice work, I have one suggestion
1. Instead of `.primary-buttons` you could use `button or btn ` class for better understanding.

Reply:
Thank you ill work on that

Reply:
Great, now it's easy to understand